borg/testsuite/convert.py

```diff
@@ -0,0 +1,97 @@
+import os
+import pytest
+import shutil
+import tempfile
+
+try:
+    import attic.repository
+    import attic.key
+    import attic.helpers
+except ImportError:
+    attic = None
+pytestmark = pytest.mark.skipif(attic is None,
+                                reason = 'cannot find an attic install')
+
+from ..converter import AtticRepositoryConverter, NotImplementedException, AtticKeyfileKey
+from ..helpers import get_keys_dir
+from ..key import KeyfileKey
+from ..repository import Repository, MAGIC
+from . import BaseTestCase
+
+class ConversionTestCase(BaseTestCase):
```

Comment:
what do you mean - with the decorator stuff? no, because we need to skip the whole module...